src/styles/blocks/catalog.scss

```
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $baseEffectDuration;
```

Specify transition properties 
```suggestion
    transition: transform $baseEffectDuration, ... ;
```